app/controllers/concerns/first_step_form_concerns.rb

```diff
@@ -0,0 +1,20 @@
+# Methods shared by controllers which can serve Step 1
+# HiringStaff::Vacancies::JobSpecificationController is school-level users' Step 1
+module FirstStepFormConcerns
```

Comment:
This module isn't a concern, should it be named differently (and moved somewhere else)?

Reply:
I wonder where to. @cesidio resident Rubyist, got any opinions?

Reply:
I'm putting the methods in application_controller
(I couldn't sleep)